scripts/quests/adoulin/a_certain_substitute_patrolman.lua

```diff
@@ -68,17 +71,14 @@ this_quest.npcs =
         ["Rising_Solstice"] =
         {
             onTrigger = function(player, npc)
-                local ACSP = player:getQuestStatus(ADOULIN, A_CERTAIN_SUBSTITUTE_PATROLMAN)
-                if ACSP == QUEST_ACCEPTED then
-                    if dsp.quests.getStage(player, this_quest) >= 1 then
-                        if dsp.quests.getStage(player, this_quest) == 8 then
-                            player:startEvent(2552) -- Finishes Quest: 'A Certain Substitute Patrolman'
-                        else
-                            player:startEvent(2551) -- Dialogue during Quest: 'A Certain Substitute Patrolman'
-                        end
-                        return true
+                if dsp.quests.getStage(player, this_quest) >= 1 then
```

Comment:
for these 'magic numbers' as i called them previously, we can probably get away with a local define in the script for the stage, though we may want a callback on `setStage` to clear any vars from the previous one

Reply:
Are we doing non-stage vars per stage? I was going to have completeQuest nuke all left-over vars (that aren't set to be preserved) just to make sure all the vars have been cleaned up.